package.json

```diff
@@ -32,8 +32,11 @@
   "dependencies": {
     "canonical-json": "0.0.4",
     "csv-parse": "^3.1.3",
+    "fs-extra": "^8.0.1",
     "googleapis": "^34.0.0",
+    "inquirer": "^6.3.1",
```

Comment:
Is this library appreciably better than `readline-sync` which is what we use in other files for prompting for user input?

Reply:
Looks like you aren't using the lib anymore, but it is still being added in the package.json